dial.go

```diff
@@ -68,7 +70,13 @@ func (opts *DialOptions) getOpts() ([]grpc.DialOption, error) {
 		return nil, err
 	}
 
-	dialOpts := []grpc.DialOption{}
+	dialOpts := []grpc.DialOption{
+		// Even though we use the default keep alive time of infinity
```

Comment:
Can you add a TODO with a ticket to re-investigate if this is still necessary during/after PM-2158?